actions/actions.py

```diff
@@ -475,7 +486,21 @@ def submit(
             after all required slots are filled"""
 
         # utter submit template
-        dispatcher.utter_message(template="utter_submit")
+        data = partial(
+            dict,
+            [
+                (slot, tracker.get_slot(slot))
+                for slot in self.SLOTS
+                if slot.startswith("symptoms_")
+            ]
+            + [
+                ("exposure", tracker.get_slot("exposure")),
+                ("age", tracker.get_slot("age")),
+            ],
+        )()
```

Comment:
I'm not sure why we're creating a `partial`, but then executing it immediately. Could we not just directly call `dict`?